source/coordinate_systems/cartesian.cc

```diff
@@ -83,6 +83,13 @@ namespace WorldBuilder
       return point_at_depth.norm();
     }
 
+
+    double
+    Cartesian::max_model_depth() const
+    {
+      return std::numeric_limits<double>::infinity();
+    }
```

Comment:
Out of curiosity, why infinity? Does the model know anything about its lateral extent?

I'm asking because you've in essence described a bounded domain (a sphere of known radius) in the spherical case, but this corresponds to an unbounded domain.

Reply:
My idea was that to fully define a position in a sphere based on depth, need a radius. For a Cartesian models that is not needed. You can go to infinity in any direction without any problem. 

I guess I could make this an option anyway within the world builder for optimization reasons, or if you really want to cut it off at some depth (which you can already do). Is that what you mean?

Reply:
Right, I understand the need for spherical objects. I was mostly wondering how you conceptually think of Cartesian worlds: do they have lateral extents that are defined somewhere, or are they infinite in the lateral directions? If the former, then it's worthwhile also specifying a maximal depth; in the latter case, there is no need for that.

Reply:
There are no lateral extends defined in Cartesian coordinates, the domain is infinite.

Reply:
I see. In that case, this makes sense.